genesis/src/lockups.rs

```diff
@@ -0,0 +1,160 @@
+//! lockups generator
+use solana_sdk::{clock::Epoch, epoch_schedule::EpochSchedule, timing::years_as_slots};
+use std::time::Duration;
+
+#[derive(Debug, Default, Clone)]
+pub struct Lockups {
+    /// where in iteration progress I am
+    expiration: usize,
+    /// number of expirations post cliff
+    expirations: usize,
+    /// fractionage offered to f() last time
+    prev_fraction: f64,
+    /// portion of total available at cliff
+    cliff_fraction: f64,
+    /// time of cliff, in epochs
+    cliff_epoch: Epoch,
+    /// number of post-cliff events
+    expiration_fraction: f64,
```

Comment:
"number of" in comment, but a f64 type?

Reply:
first implementation was number of events, comment is wrong